tests/foreman/cli/test_logging.py

```diff
@@ -29,6 +29,8 @@
 from robottelo.config import settings
 from robottelo.logging import logger
 
+pytestmark = pytest.mark.e2e
```

Comment:
I don't feel these are end to end tests

Reply:
These together marks the logging of different components.

Reply:
Okay I think for this component we can make this as e2e.